### PR TITLE
slint, travis set up

### PR DESCRIPTION
**Pull request by amboraes1**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            "cypress"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (manuelq12) [inline tsconfig.json]: Missed EOF

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v14.19.0
```
Comment (manuelq12) [inline .nvmrc]: EOF

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: .nvmrc (hunk 2 above)
Comment (amboraes1) [reply .nvmrc]: Changes added

File: tsconfig.json (hunk 1 above)
Comment (amboraes1) [reply tsconfig.json]: Changes added

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**